test/Microsoft.VisualStudio.LanguageServices.Razor.Test/RazorEditorParserTest.cs

```diff
@@ -85,7 +85,12 @@ public static TheoryData TagHelperPartialParseRejectData
 
         [Theory]
         [MemberData(nameof(TagHelperPartialParseRejectData))]
-        public void TagHelperTagBodiesRejectPartialChanges(object editObject, object expectedDocument)
+        public void TagHelperTagBodiesRejectPartialChanges(object editObject,
+            // This parameter is not used in this tests. This disables the build warning
+            // that says 'Theory methods should use all of their parameters'
+#pragma warning disable xUnit1026
+            object expectedDocument)
```

Comment:
Instead of suppressing, would it be better to split the MemberData so that it doesn't result in used theory params?